src/dma.rs

```diff
@@ -90,6 +90,22 @@ impl DmaBuf {
     pub fn to_io_vec(&self) -> IoVec {
         self.0
     }
+
+    /// Helper to dump content of a DmaBuf.
+    /// Derived Debug trait for DmaBuf only prints base address and length.
+    pub fn dump(&self) {
```

Comment:
Should this return a String and allow the caller to use whatever print kind it wants?

Reply:
Yeah, that can be done.